cli/src/main.rs

```diff
@@ -22,7 +23,11 @@ async fn get_dockerfile_b64() -> String {
       return base64::encode(dockerfile);
     }
     Err(_) => {
-      error!(100, "Current working directory value is invalid").await;
+      error!(
+        ErrorKind::InvalidPwd as u8,
```

Comment:
Is there any way we can avoid all of this casting? @cdmistman?

Reply:
Already updated to avoid casting ;)